ci-operator/config/openshift-knative/serving/openshift-knative-serving-release-next__411.yaml

```diff
@@ -319,34 +150,47 @@ tests:
           cpu: 100m
       timeout: 4h0m0s
     workflow: generic-claim
-- as: e2e-aws-ocp-tls-411
```

Comment:
We probably want to keep the `tls` job as a separate config, right?

Reply:
Yes, that's right.